base/docs/basedocs.jl

```diff
@@ -1967,6 +1981,21 @@ Signed
     Unsigned <: Integer
 
 Abstract supertype for all unsigned integers.
+
+All are printed in hexadecimal, with prefix `0x`,
+and can be entered in the same way.
```

Comment:
This is true for all subtypes of `Unsigned` defined in Base but does not necessarily hold for subtypes defined in packages.

Reply:
It could say something like this:
```suggestion
All built-in unsigned integers printed in hexadecimal, with prefix `0x`,
and can be entered in the same way.
```
Are there any defined elsewhere which don't obey this? Defining a new unsigned integer type sounds like an odd thing to do.

Reply:
SaferIntegers.jl defines `SafeUInt8 <: SafeUnsgined <: Unsgined` which errors on overflow. These are printed in hexadecimal, but cannot be entered in the same way
```julia
julia> x = SafeUnsigned(0x17)
0x17

julia> x === 0x17
false
```
Perhaps drop the "all"
```suggestion
Built-in unsigned integers are printed in hexadecimal, with prefix `0x`,
and can be entered in the same way.
```

Reply:
Done.